api/api/views.py

```diff
@@ -2,7 +2,9 @@
 from datetime import date
 
 from datapunt_api.rest import DatapuntViewSet
-from django.http import Http404, HttpResponse, HttpResponseServerError, StreamingHttpResponse
+from django.conf import settings
+from django.http import Http404, HttpResponse, HttpResponseServerError
+from django.http import StreamingHttpResponse
```

Comment:
Is importing these on separate lines done because of flake8?

Reply:
Nope. Flake is quite mad about it actually. Fixing now.